cypress/e2e/models/migration/reports/reports.ts

```diff
@@ -16,57 +16,40 @@ limitations under the License.
 /// <reference types="cypress" />
 /// <reference types="cypress-xpath" />
 
-import { applicationName, risk } from "../../../types/constants";
+import { selectItemsPerPage } from "../../../../utils/utils";
+import { SEC, applicationName, risk } from "../../../types/constants";
 import * as commonView from "../../../views/common.view";
-import { itemsPerPageMenu, itemsPerPageToggleButton } from "../../../views/reports.view";
+import {
+    adoptionCandidateDistributionTitle,
```

Comment:
This is not used anywhere. Please remove

Reply:
done